cluster-operator/src/test/java/io/strimzi/operator/cluster/model/EntityOperatorTest.java

```diff
@@ -294,6 +296,12 @@ public void testTemplate() {
                                     .withAnnotations(saAnnotations)
                                 .endMetadata()
                             .endServiceAccount()
+                            .withNewPodDisruptionBudget()
+                                .withNewMetadata()
+                                    .withLabels(Map.of("pdb-label", "pdb-value"))
+                                    .withAnnotations(Map.of("pdb-annotation", "pdb-value"))
+                                .endMetadata()
+                            .endPodDisruptionBudget()
```

Comment:
This test should probably have some asserts as well?

Also, Maybe we can follow the same pattern as we do for the other resources and define the labels/annotation maps up-front? (See the Service Account 4 lines above) ... that would make the maintenance in the future easier as it would be all done in the same way and all together.